src/configure/helper/redirectLinkHelper.ts

```diff
@@ -0,0 +1,38 @@
+import { RestClient } from "typed-rest-client";
+
+export enum ServiceFramework {
+    Moda,
+    Vssf
+}
+
+export class RedirectLinkHelper {
+
+    public repoAnalysisRedirectUrl: string  = "https://aka.ms/AA87j8k";
```

Comment:
should we not use a fwd link instead of aka.ms?

Reply:
I couldn't find any reason why fwd link could be better, shall we sent out a mail to the owner team for their opinion? It'll help us understand if there are any security/compliance issues as well in using their product for our purpose.